uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/WorkDayTest.php

```diff
@@ -156,16 +159,38 @@ public function testCannotHaveMoreThanOfDayOfTheWeekForEachWeek() {
   }
 
   /**
-   * @dataProvider numberOfHoursDataProvider
+   * @dataProvider validNumberOfHoursDataProvider
    */
-  public function testNumberOfHoursShouldBeCalculatedFromTimesAndBreak($timeFrom, $timeTo, $break, $expectedNumberOfHours) {
-    $entity = $this->createBasicWorkDay([
-      'time_from' => $timeFrom,
-      'time_to' => $timeTo,
-      'break' => $break
-    ]);
+  public function testNumberOfHours($numberOfHours) {
```

Comment:
This test name needs a bit more description, e.g `testWorkDatsCanBeCreatedWithValidHours`.

Reply:
Done.